tests/covalent_tests/workflow/transport_test.py

```diff
@@ -127,6 +115,29 @@ def test_transportable_object_get_deserialized(transportable_object):
     assert to.get_deserialized()(x=2) == subtask(x=2)
 
 
+def test_transportable_object_sedeser_string_only():
+    """Test extracting string only from serialized to"""
+    x = 123
+    to = TransportableObject(x)
+
+    ser = to.serialize()
+    new_to = TransportableObject.deserialize(ser, string_only=True)
+    assert new_to.object_string == to.object_string
+    assert new_to._object == ""
+
+
+def test_transportable_object_sedeser_header_only():
+    """Test extracting header only from serialized to"""
+    x = 123
+    to = TransportableObject(x)
+
+    ser = to.serialize()
+    new_to = TransportableObject.deserialize(ser, header_only=True)
+
+    assert new_to.object_string == ""
+    assert new_to._header
```

Comment:
I think there should be a comparison here. For instance check the python version.

Reply:
Yezzir